interface/context/context.go

```diff
@@ -1,4 +1,4 @@
-package controller
+package context
```

Comment:
There is already a go package `context`, you would probably like to name this differently to avoid conflicts

https://pkg.go.dev/context